src/Site.js

```diff
@@ -813,6 +815,19 @@ Site.prototype.updateSiteData = function (filePaths) {
   this.writeSiteData();
 };
 
+/**
+ * Copies Font Awesome assets to the assets folder
+ */
+Site.prototype.copyFontAwesomeAsset = function () {
+  const faRootSrcPath = path.join(__dirname, '..', 'node_modules', '@fortawesome', 'fontawesome-free');
+  const faCssSrcPath = path.join(faRootSrcPath, 'css', 'all.min.css');
```

Comment:
The package itself also provides `font-awesome.min.css`, so is there a reason why we use `all.min.css` instead of `font-awesome.min.css`?

Reply:
`fontawesome.min.css` is meant to be used in combination with the other CSS files in `css/` (`brands.css`, `solid.css`, `regular.css`, `light.css`) [to enable only specific icon styles](https://fontawesome.com/how-to-use/on-the-web/setup/hosting-font-awesome-yourself#using-certain-styles). 

Since we are enabling all icon styles, we should use `all.min.css`, which [enables all icon styles in a single file](https://fontawesome.com/how-to-use/on-the-web/setup/hosting-font-awesome-yourself#using-web-fonts).

Reply:
Hmm, is it misleading to call the final destination css to be `fontawesome.min.css` then? 🤔 Should we also change it to `all.min.css` for the final destination css?

Reply:
I think the original reason for naming it `font-awesome.min.css` was to not have a `all.min.css` in the `css/` folder, since it would be confusing.

However, now that we have moved FA to `fontawesome/`, `fontawesome/css/all.min.css` would be less confusing. I will retain the original `all.min.css` naming then.